nifi-nar-bundles/nifi-cassandra-bundle/nifi-cassandra-processors/src/main/java/org/apache/nifi/processors/cassandra/PutCassandraRecord.java

```diff
@@ -67,6 +75,35 @@
             .required(true)
             .build();
 
+    static final PropertyDescriptor STATEMENT_TYPE = new PropertyDescriptor.Builder()
+            .name("put-cassandra-record-statement-type")
+            .displayName("Statement Type")
+            .description("Specifies the type of CQL Statement to generate.")
+            .required(true)
+            .defaultValue(INSERT_TYPE)
+            .allowableValues(UPDATE_TYPE, INSERT_TYPE)
+            .build();
+
+    static final PropertyDescriptor UPDATE_METHOD = new PropertyDescriptor.Builder()
+            .name("put-cassandra-record-update-method")
+            .displayName("Update Method")
+            .description("Specifies the method to use to SET the values.")
+            .required(false)
+            .defaultValue(SET_TYPE)
+            .allowableValues(INCR_TYPE, DECR_TYPE, SET_TYPE)
+            .build();
+
+    static final PropertyDescriptor UPDATE_KEYS = new PropertyDescriptor.Builder()
+            .name("put-cassandra-record-update-keys")
+            .displayName("Update Keys")
+            .description("A comma-separated list of column names that uniquely identifies a row in the database for UPDATE statements. "
+                    + "If the Statement Type is UPDATE and this property is not set, the conversion to CQL will fail. "
+                    + "This property is ignored if the Statement Type is INSERT.")
```

Comment:
Nitpick, but in case more verbs are added later, I think the doc should read "This property is ignored if the Statement Type is anything other than UPDATE"